openssl-dynamic/src/test/java/io/netty/internal/tcnative/CertificateVerifierTest.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.internal.tcnative;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+
+public class CertificateVerifierTest extends AbstractNativeTest {
+
+    @Test
+    public void testValidErrorCode() throws Exception {
+        Field[] fields = CertificateVerifier.class.getFields();
+        for (Field field : fields) {
+            if (field.isAccessible()) {
+                int errorCode = field.getInt(null);
+                Assert.assertTrue(CertificateVerifier.isValid(errorCode));
+            }
+        }
+    }
+
+    @Test
+    public void testNonValidErrorCode() {
+        // Will overflow on first increment
+        int errorCode = Integer.MAX_VALUE;
+        do {
+            errorCode++;
+            Assert.assertTrue(errorCode < Integer.MAX_VALUE);
+        } while (CertificateVerifier.isValid(errorCode));
```

Comment:
yeah that it not always return true. Maybe using Integer.MIN_VALUE is good enough as you said.